src/common/post.ts

```diff
@@ -49,3 +50,17 @@ export const getPostCommenterIds = async (
 
 export const hasAuthorScout = (post: Post): boolean =>
   !!post?.authorId || !!post?.scoutId;
+
+export const linkPreviewOrigin = process.env.LINK_PREVIEW_ORIGIN;
+
+export const fetchLinkPreview = async (
+  url: string,
+): Promise<ExternalLinkPreview> => {
+  const res = await fetch(`${linkPreviewOrigin}/preview`, {
```

Comment:
What happens when there is a error here?
Thinking we should maybe try/catch this in case? (+ maybe add log on the fail)

Reply:
`fetch` by default does not throw on non `200` status codes, so probably good to add `res.ok` check and then handle/log.

Reply:
Oh, right. Let me throw an error when that happens.